### PR TITLE
[bn]: Translation update

### PR DESCRIPTION
**Pull request by arman-arif**

### CODE REVIEW

File: locales/bn/php-inline.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
     "required": "এই ক্ষেত্রটি প্রয়োজন.",
     "required_array_keys": "এই ক্ষেত্রটিতে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে",
     "required_if": "এই ক্ষেত্র প্রয়োজন বোধ করা হয় যখন :other হয় :value.",
-    "required_if_accepted": "This field is required when :other is accepted.",
+    "required_if_accepted": "এই ক্ষেত্রটি প্রয়োজন হয় যখন :other গৃহীত হয়",
```
Comment (andrey-helldar) [inline locales/bn/php-inline.json:94]:
```suggestion
    "required_if_accepted": "এই ক্ষেত্রটি প্রয়োজন হয় যখন :other গৃহীত হয়.",
```

File: locales/bn/php.json [hunk 2]
```diff
@@ -96,7 +96,7 @@
     "required": ":Attribute স্থানটি পূরণ করা বাধ্যতামূলক।",
     "required_array_keys": ":Attribute ফিল্ডে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে",
     "required_if": ":Attribute স্থানটি পূরণ করা বাধ্যতামূলক যেখানে :other হল :value।",
-    "required_if_accepted": "The :attribute field is required when :other is accepted.",
+    "required_if_accepted": ":attribute ক্ষেত্রটি প্রয়োজন হয় যখন :other গৃহীত হয়",
```
Comment (andrey-helldar) [inline locales/bn/php.json:99]:
```suggestion
    "required_if_accepted": ":attribute ক্ষেত্রটি প্রয়োজন হয় যখন :other গৃহীত হয়.",
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/bn/php-inline.json [hunk 3]
```diff
@@ -91,7 +91,7 @@
     "required": "এই ক্ষেত্রটি প্রয়োজন.",
     "required_array_keys": "এই ক্ষেত্রটিতে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে",
```
Comment (andrey-helldar) [inline locales/bn/php-inline.json:92]:
```suggestion
    "required_array_keys": "এই ক্ষেত্রটিতে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে.",
```

File: locales/bn/php.json [hunk 4]
```diff
@@ -96,7 +96,7 @@
     "required": ":Attribute স্থানটি পূরণ করা বাধ্যতামূলক।",
     "required_array_keys": ":Attribute ফিল্ডে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে",
```
Comment (andrey-helldar) [inline locales/bn/php.json:97]:
```suggestion
    "required_array_keys": ":Attribute ফিল্ডে অবশ্যই :values এর জন্য এন্ট্রি থাকতে হবে.",
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
In some places sentences end with a full stop (`.`), and in others with a slash (`|`).

Which option is correct?

File: locales/bn/php-inline.json (hunk 3 above)
Comment (arman-arif) [reply locales/bn/php-inline.json:92]: In bengali (.) = (।)

**Comment by arman-arif** [comment]
In Bengali  (`.`) = (`।`)

I will change all `.` and push a commit. Please wait until that.

**Comment by arman-arif** [comment]
Made some correction and all (`.`) at the end of sentences are now changed to (`। `)

**Comment by andrey-helldar** [comment]
@arman-arif, understood, thanks 🤝

**Review by andrey-helldar — APPROVED** [review]
তোমাকে ধন্যবাদ

**Comment by arman-arif** [comment]
> তোমাকে ধন্যবাদ

স্বাগতম

**Pull request merged**